From 516f8b15603b7f7613e2fb957c55bc56a36b64a6 Mon Sep 17 00:00:00 2001
From: Keith Smiley <keithbsmiley@gmail.com>
Date: Wed, 28 Aug 2019 21:41:02 -0700
Subject: [PATCH] bazel: Change zlib URL to GitHub

We've notice significant issues downloading from zlib.net. Since there
are already other archives coming from GitHub, this shouldn't negatively
affect reliability.
---
 protobuf_deps.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/protobuf_deps.bzl b/protobuf_deps.bzl
index e5117d167d..9209b1c5a3 100644
--- a/protobuf_deps.bzl
+++ b/protobuf_deps.bzl
@@ -17,7 +17,7 @@ def protobuf_deps():
         http_archive(
             name = "zlib",
             build_file = "@com_google_protobuf//:third_party/zlib.BUILD",
-            sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1",
+            sha256 = "629380c90a77b964d896ed37163f5c3a34f6e6d897311f1df2a7016355c45eff",
             strip_prefix = "zlib-1.2.11",
-            urls = ["https://zlib.net/zlib-1.2.11.tar.gz"],
+            urls = ["https://github.com/madler/zlib/archive/v1.2.11.tar.gz"],
         )

     if not native.existing_rule("six"):
